cartridges/int_alma/cartridge/scripts/helpers/almaCheckoutHelper.js

```diff
@@ -162,6 +166,10 @@ function isFragmentActivated() {
  * @returns {array} a plan Object understandable for Eligibility
  */
 function formatPlanForCheckout(plan, currencyCode) {
+    var logger = require('dw/system/Logger');
+    logger.error(plan.purchaseAmount.toString());
```

Comment:
why we have an `error` logged here ?

if log is required, please add some context, not just the `purchaseAmount` as string 😉

Reply:
because I've left my debug :( thanks for catching that ... )